Bottom: 7d9cc9b0e7d8e2ede41adff45fcaf4b3c2c1c77e
Top:    ea36b1ae7e5cb3337f674c7be28ac46369d4bb20
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 21:59:02 +0000

Revert "Test routine for poly_InvContour"

This reverts commit e1de240da219e7cd5696ed8dde04b0483abd6e54.




---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3442cca..e3ed7b8 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2879,8 +2879,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void test_polyInvContour (void);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
@@ -2893,8 +2891,6 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   CVCList *the_list;
   POLYAREA *a_copy, *b_copy;
 
-  test_polyInvContour ();
-
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
@@ -3276,32 +3272,6 @@ flip_cb (const BoxType * b, void *cl)
   return 1;
 }
 
-static void
-test_polyInvContour (void)
-{
-  static bool done = false;
-  PLINE *contour;
-  Vector v;
-
-  if (done)
-    return;
-
-  printf ("Testing inv_contour\n");
-
-  v[0] = 0, v[1] = 0;           contour = poly_NewContour (poly_CreateNodeArcApproximation (v, 0, 0, 1));
-  v[0] = 0, v[1] = 2; poly_InclVertex (contour->head.prev, poly_CreateNodeArcApproximation (v, 0, 0, 3));
-  v[0] = 0, v[1] = 4; poly_InclVertex (contour->head.prev, poly_CreateNodeArcApproximation (v, 0, 0, 5));
-  v[0] = 0; v[1] = 6; poly_InclVertex (contour->head.prev, poly_CreateNodeArcApproximation (v, 0, 0, 7));
-
-  pline_dump (&contour->head);
-  poly_InvContour (contour);
-  pline_dump (&contour->head);
-
-  poly_FreeContours (&contour);
-
-  done = true;
-}
-
 void
 poly_InvContour (PLINE * c)
 {
